Description: Bump hex-literal dependency
 The new version does not break compatibility with usage here
Author: Antonio Russo <aerusso@aerusso.net>
Forwarded: not-needed
Last-Update: 2023-08-23
Index: blake2/Cargo.toml
===================================================================
--- blake2.orig/Cargo.toml
+++ blake2/Cargo.toml
@@ -39,7 +39,7 @@ version = "0.10.3"
 features = ["dev"]
 
 [dev-dependencies.hex-literal]
-version = "0.2.2"
+version = "0.4.1"
 
 [features]
 default = ["std"]
